src/js/components/Text/Text.js

```diff
@@ -70,10 +71,10 @@ const Text = forwardRef(
     const styledTextResult = (
       <StyledText
         as={!as && tag ? tag : as}
-        colorProp={color}
         aria-label={a11yTitle}
-        truncate={truncate}
-        size={size}
+        $colorProp={color}
```

Comment:
```suggestion
        $color={color}
```
Should we just rename this to `$color`?

Reply:
For some reason I was getting a type error on that one:
<img width="950" alt="Screen Shot 2023-07-11 at 1 28 45 PM" src="https://github.com/grommet/grommet/assets/12522275/47991a02-9c49-488e-9e7b-5e5356b8cd5d">

Reply:
Not sure if it's a special case?

Reply:
Hmm interesting, in that case I'm fine with leaving it as `colorProp` for now